frontend/views/utils/currencies.js

```diff
@@ -7,8 +7,38 @@
 // this value, switch to a different currency base, e.g. from BTC to mBTC.
 export const DECIMALS_MAX = 8
 
-export function saferFloat (value: number): number {
-  return parseFloat(value.toFixed(DECIMALS_MAX))
+function commaToDots (value) {
+  // ex: "1,55" -> "1.55"
+  return typeof value === 'string' ? value.replace(/,/, '.') : value
```

Comment:
Just a quick question about this: is there ever a time this wouldn't be a string? It's hard to trace all the code that calls it, but it always seems to be either a string or a number. Is it expected to ever be undefined or null?

Reply:
I want to believe it's always a string or a number :D. I don't remember being otherwise.